### ISSUE TITLE
[Windows] Build error

### ISSUE THREAD

**Issue by derevandal** [mustardamus/nuxt-bulma-slim#1]
Hi,
I'm on W10 x64 with node v8.9.4 and yarn v1.5.1.
I tried use this module and I got this error:
```posh
 ERROR  Failed to compile with 1 errors                                 17:21:38
This relative module was not found:
* ./C:\Users\derev\AppData\Local\Temp\nuxt-bulma-slim.sass in ./.nuxt/App.js
  nuxt: Call close hooks (2) +4s
  nuxt: server closed +2ms
 OPEN  http://localhost:3000
```
So...
```javascript
  modules: [
    '@nuxtjs/pwa',
    [
      'nuxt-bulma-slim',
      {
        sassTempPath: './tmp/nuxt-bulma-slim.sass'
      }
    ]
  ]
```
I changed my nuxt.config.js to customize sassTempPath and  I got this error too:
```posh
 ERROR  Failed to compile with 1 errors                                 17:26:27

 error  in ./tmp/nuxt-bulma-slim.sass

Module build failed:
@import "D:\dev\lenyeg\cruzeirovirtual\node_modules\bulma\bulma.sass"
^
      File to import not found or unreadable: D:Þvlenyeg
                                                        ruzeirovirtualnode_modu
les
      File to import not found or unreadable: D:Þvlenyeg
       ulma.sass.                                       ruzeirovirtualnode_modules
Parent style sheet: stdin                                                         ulma
      in D:\dev\lenyeg\cruzeirovirtual\tmp\nuxt-bulma-slim.sass (line 1, column       ulma.sass.
Parent style sheet: stdin
      in D:\dev\lenyeg\cruzeirovirtual\tmp\nuxt-bulma-slim.sass (line 1, column 1)

 @ ./tmp/nuxt-bulma-slim.sass 4:14-201 13:3-17:5 14:22-209
 @ ./.nuxt/App.js
 @ ./.nuxt/index.js
 @ ./.nuxt/client.js
 @ multi webpack-hot-middleware/client?name=client&reload=true&timeout=30000&path=/__webpack_hmr ./.nuxt/client.js

 OPEN  http://localhost:3000
```

Could u help me?

**Comment by mustardamus** [comment]
Hey André, thanks for checking out this module. I haven't had a chance to test it on Windows, so lets figure this out.

So with the default value, the leading `./` in the path must be the problem. Seems like the path to the temp .sass isn't treated as an absolute path. Unfortunately I haven't found related Nuxt.js issues. What I can do is to convert the absolute paths back to relatives.

I guess thats also the same problem with the second problem, it can not resolve the absolute path.

So I'll make a fix to only use relative paths. Probably today or tomorrow. Would you help me test it?

**Comment by frederic117** [comment]
i got exactly the same problem !!!

**Comment by kfrederix** [comment]
Had the same errors.
Made some small changes in my fork which fix it for me: https://github.com/kfrederix/nuxt-bulma-slim
Also opened a PR.

**Comment by mustardamus** [comment]
Merged in @kfrederix code and published version 0.3.1 to NPM. Still works for me on *nix, but still no way to test on a Windows machine. So if anybody runs into problems on Windows, let me know.
